testcase/stem_middle.mk

```diff
@@ -0,0 +1,11 @@
+
+test: a/a.a b.b c/c
```

Comment:
Can you make this test fail at HEAD?

Reply:
It does

Reply:
just as embarrassing (for me) as on the other thread